src/verify/VerificationCodeInputScreen.test.tsx

```diff
@@ -131,6 +126,38 @@ describe('VerificationCodeInputScreen', () => {
     expect(navigate).toHaveBeenCalledWith(Screens.OnboardingSuccessScreen)
   })
 
+  it('waits for the verificationId to be captured before verifying sms', async () => {
+    mockFetch.mockImplementation(async (url?: string | Request) => {
+      if (url === networkConfig.verifyPhoneNumberUrl) {
+        await sleep(1000) // some arbitrary network delay
```

Comment:
nit: does this increase this unit test run by 1 sec, or are advancing timers to avoid this?

Reply:
we're advancing timers :D this test doesn't use real timers